### ISSUE TITLE
Remarks for portfolio v2

### ISSUE THREAD

**Issue by sirMerr** [xpack94/portfolioV2#2]
Yay gh-pages c:. Few things. 

## Code & Structure
* You have `portfolioCss.css` and `portfoliojs.js`. Simply rename it to `style.css` and `index.js` (names don't need to be this). The `.css` ending makes it clear it's a CSS file, no need to repeat it.
* For your typing animation, I see you're using  `typed.js`. If you want to learn how to do this yourself, you can check out https://css-tricks.com/snippets/css/typewriter-effect/ . 
* In your [CSS](https://github.com/xpack94/portfolioV2/blob/gh-pages/portfolioCss.css#L30) and your HTML, you're using a GitHub link to images that are in your repository. This is not the way to do this. Try and find a way to use relative paths instead.
* Your [`index.html`](https://github.com/xpack94/portfolioV2/blob/gh-pages/index.html#L24) has what I assume is a default title. Change it to your own and look up what that tag and other meta tags are for.
* Your indentation is off and you have empty new lines everywhere. Use a linter basically 😄.
* I'm not sure why you have an additional script in your html for your `typed.js` integration. Have that be in your `.js` file. 
* You still have a `console.log` in your js.
* Your `portfoliojs.js` file should be at the beginning of your HTML with the rest of your script tags. In your `js' file, you already have `$(function() {})`, so putting the script at the end is redundant. Look up what `$(function() {})`, `DOM` and `DOMContentLoaded` are, because I believe you might not understand properly what these do. JQuery is fun and all, but you have to understand that behind that is javascript and you should understand how the jQuery functions you use work. Sometimes you use `querySelector()` and `addEventListener`, sometimes you use the equivalent with `$()`. There is a lot of inconsistencies with your code.
* Kind of goes with my above point, but **you should not have all your code inside `$function() {}`**. Once you understand what it's used for, you should understand what goes in and out of it.

## Design & Typos
* Design wise, you still have too many fonts in my opinion, but that's up to you.
* Your arrow is not centered and still not smooth. Once it hits the bottom of your div, it jumps back up to the original position, making it clunky.
* Your form does not have proper capitalization. If you are more comfortable with French, you could write in French, or provide a french version of your portfolio ^-^.
* Your text is still hard to read, and clunky. I suggest you get someone to proofread and edit the text with you.

## All in All
I think you have some misunderstandings about file structures and paths. I think you also do not clearly understand how to use JavaScript and JQuery, nor why and how certain functions work and why you would use one versus another. Additionally, your indentations and separation of code are very lacking. It may be better for you to just write your code in vanilla JavaScript, until you have a better understanding of what you're using.

**Comment by xpack94** [comment]
thank you for your incredible feedback as always it helps me improve.
when it comes to js and jquery is it okay if for example i declare all the variables using jquery inside the $(function) and use plain javascript to implement the portfolio or does make seem inconsistent? 
for the fonts i think i used two fonts it's just that some of them have font-weight higher than others 
i will try to improve the code in the upcoming days 
thanks

**Comment by sirMerr** [comment]
That may be for the fonts, I didn't look too closely. It seemed like a lot of styles.

So jQuery is a library. It is not a different language, it's just like you using `typed.js, except it's a lot bigger and does different things. It's used to simplify common javascript functions and more.

The `$(function)` part is used to run code only when the DOM has been loaded, which is the equivalent of you having the script tag at the bottom of your HTML (which means it's redundant the way you did it). So method definitions should not be in there. `$(function)` **is** jQuery. Anything using `$` is jQuery. By default, jQuery uses "$" as a shortcut for "jQuery". 

So `$(function)`, is the same as `document.addEventListener("DOMContentLoaded", function())` in plain javascript (which means without jQuery or any other libraries).

Check here for more information: https://jquery.com/

**Comment by xpack94** [comment]
yeah i already knew all these concepts but sometimes when i don't know how to do something in plain javascript i do in in jquery , or if it's gonna be long in js i do it in jquery that's why i mix both of them together .
i put the typed.js in the html because at the beginning i wasn't sure that i was gonna use it so i was experimenting with it there , and then i forgot it xd 
but your comments and feedback help me grow and improve thanks alot

**Comment by sirMerr** [comment]
No problem, sorry I'm being redundant with stuff you already know xD. Yeah jquery adds a lot of shorthand to common methods, but sometimes in your code, you use the default functions like `querySelector()` instead of using jQuery's `$()` to do the same thing. There's nothing wrong using one or the other, but if you're going to be using a type of function to do something, use it everywhere for consistency is what I mean.

FYI, using a code editor like VSCode will help with long functions since they have auto-suggestion and auto-complete.

**Comment by xpack94** [comment]
oh not at all and thanks for the vsCode imma check it out asap .
i will be posting my project on the freecodecamp group so that you can gimme your feedback because they have helped a lot 
thanks again
